Peer.Domain/Models/UserClient.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Octokit;
+
+namespace Peer.Domain.Models
+{
+    public class UserClient
```

Comment:
Any specific reason you're doing this through a UserClient class? Doesn't seem to hide any deps since it exposes the Octokit client and doesn't add any custom behaviour